kafka/protocol/metadata.py

```diff
@@ -52,17 +52,15 @@ class MetadataRequest_v0(Struct):
     API_VERSION = 0
     RESPONSE_TYPE = MetadataResponse_v0
     SCHEMA = Schema(
-        ('topics', Array(String('utf-8'))) # Empty Array (len 0) for all topics
+        ('topics', Array(String('utf-8')))  # Empty Array (len 0) returns all topics
     )
 
 
 class MetadataRequest_v1(Struct):
     API_KEY = 3
     API_VERSION = 1
     RESPONSE_TYPE = MetadataResponse_v1
-    SCHEMA = Schema(
-        ('topics', Array(String('utf-8'))) # Null Array (len -1) for all topics
-    )
+    SCHEMA = MetadataRequest_v0.SCHEMA  # Null Array (len -1) returns all topics
```

Comment:
Pushed an updated version that includes these class vars. I'm still not sure what correct values are for empty arrays in this structs class, so I set them to `None`, but perhaps they should be `0`.